src/shared/utils.ts

```diff
@@ -5,7 +5,7 @@ import * as globby from 'globby';
 import * as mmd from 'music-metadata';
 import pickBy from 'lodash-es/pickBy';
 
-import { Track } from '../../shared/types/interfaces';
+import { Track } from './types/interfaces';
```

Comment:
I think there was a misunderstanding. This file should stay where it was. Only extract the `fetchCover` (and the `parseBase64` I guess) to `src/shared/utils.ts`.

(only move those two function, not the entire file)

Reply:
Oh shit. Let's fix it.